auth/src/main/java/com/google/cloud/auth/samples/AuthExample.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2018 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.auth.samples;
+
+import com.google.api.gax.paging.Page;
+import com.google.auth.oauth2.ComputeEngineCredentials;
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.cloud.storage.Bucket;
+import com.google.cloud.storage.Storage;
+import com.google.cloud.storage.StorageOptions;
+import com.google.common.base.Strings;
+import com.google.common.collect.Lists;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+
+/**
+ * Hit google cloud translate REST endpoint; uses google-auth library.
+ */
+public class AuthExample {
+  public static void main(String[] args) throws IOException {
```

Comment:
I presume this is intentional.  Our standard tends to put main() at the end, but if you think it's better in front - ie. folks actually look at it before running, then I'm ok w/ it.

Reply:
I don't have a preference, so I will move it to meet the standard.

This did make me notice that the class comment needs to be updated as well though.